### ISSUE TITLE
[Feature Request] Wiki page for integration with webhooks

### ISSUE THREAD

**Issue by joveice** [AHAAAAAAA/PokemonGo-Map#2472]
Have a wiki page for how to setup webhooks to work with etc, pushbullet or twitter to create notifications when pokemons you have added in a config file spawns. notification contains link that points on it on your map.

If someone could do this It would be awesome :)

**Comment by andrewda** [comment]
I already published one about 12 hours ago https://github.com/AHAAAAAAA/PokemonGo-Map/wiki/Using-Webhooks :)

**Comment by joveice** [comment]
@andrewda Oh nice, Maby add this to the new wiki page?

**Comment by joveice** [comment]
StPaddy81 linked me this on Discord https://github.com/kvangent/PokemonGo-Map/tree/develop

They have added this, would this be possible to get in this to?

**Comment by andrewda** [comment]
@joveice nope. The whole point of adding webhooks is to make the code as clean as possible. They also allow you to do an infinite number of things with the data that doesn't have to be hard-coded into PokemonGo-Map.

**Comment by Malouff** [comment]
Can you please give an example on how to implement? 
Say with Slack

I am using kvangent's branch & love using Slack to alert friends in real time of true or false Pokémon. 
Slack has a snooze feature too for when you are sleeping or at work. 

It would be nice if the true and false list was tied to the maps Notify of Pokémon and Hide common Pokémon (or in a settings pannel like options one) on the web server. 

Even better would be if it was arguments for a herokuapp.com deploy. 
True Pokémon argument 
False Pokémon argument 
Webhook argument to use (I am not sure how it would change the requirements.txt as I see kvangent's branch had new requirements,  that &  how you could use more than one wehook service)

**Comment by wschoot** [comment]
PHP example for slack:

```
    curl_setopt_array($ch = curl_init(), array(
    CURLOPT_URL            => 'https://hooks.slack.com/services/' . $slackhook,
    CURLOPT_CUSTOMREQUEST  => 'POST',
    CURLOPT_RETURNTRANSFER => true,
    CURLOPT_SSL_VERIFYPEER => false,
    CURLOPT_POSTFIELDS     => "payload=" . json_encode(array(
        "channel"      =>  "#pokemon",
        "text"         =>  "<$gurl|$poke ($poke_id)> until $until (".date("i\ms\s\)", $diff),
    ))));
    $result = curl_exec($ch);
    curl_close($ch);
```

I also built Pushover and Telegram support. Would dev be open to a crontrib of examples dir which I can fill with examples?

**Comment by msbullock** [comment]
@wschoot I'd love the full PHP file for the slack example above. I don't see how you are having the script listen to the runserver.py output.

**Comment by wschoot** [comment]
@msbullock See https://gist.github.com/wschoot/01f6d24c83e18ca955d75bcee2633fc4 -- You need to create a web integration into slack (which is pretty easy from their admin panel) and use the --webhook option from runserver.py (ofcourse this piece of PHP needs to run on a PHP-server which can be reached from the runserver-server)

**Comment by msbullock** [comment]
I've got my slack up and running, and I'm familiar with their commands and
they have some curl examples like what you posted. I dont know how to take
your post.php and make it run in a server. Can i just find any PHP server
example and paste the post.php code into it, or does the PHP server need to
call the post.php?

**Comment by wschoot** [comment]
Your runserver.py will call post.php on a PHP enabled webserver using the --webhook functionality.
